libs/SmartStore/src/com/salesforce/androidsdk/smartstore/store/SqliteLibraryLoader.java

```diff
@@ -0,0 +1,220 @@
+/*
```

Comment:
Change copyright to `Mobile SDK` copyright. You should be able to get it from one of the other files.